src/get-dart.md

```diff
@@ -58,6 +57,26 @@ so it's easy to find (but not more tempting than package managers).
 {% include_relative tools/sdk/_mac.md %}
 </div>
 
+## System requirements
+
+The Dart SDK is supported on the following operating systems:
+
+#### Windows
+* _Supported versions_: Windows 10.
+* _Supported architectures_: x64, ia32.
+
+#### Linux
+* _Supported versions_: Most recent Linux versions; we however only test with Ubuntu 16.04.
+* _Supported architectures_: x64, ia32, arm, arm64.
+* _Notes_: On arm, we require Glibc 2.23 or newer due to [a bug](https://sourceware.org/bugzilla/show_bug.cgi?id=14341) in the dynamic linker.
+
+#### macOS
+* _Supported versions_: Latest three major versions, currently:
```

Comment:
Would that mean that if Apple releases a new version we just update the list below and drop support for the oldest version?

Reply:
Yes, that is my suggestion. We support 3 last versions in a rolling window. Typically Apple releases one new version a year, usually in the fall.